### DISCUSSION TITLE
全球投资开户完全指南：A股与美股篇 · BMPI

### DISCUSSION THREAD

**Discussion by giscus** [bmpi-dev/bmpi.dev#156] [Announcements]
# 全球投资开户完全指南：A股与美股篇 · BMPI

本文分享如何零成本开通A股与美股的各类证券账户并开启全球投资之旅的个人经验。

https://www.bmpi.dev/money/guide-to-open-global-investment-account/

**Comment by Mistsink** [comment]
你好，请问现在怎么申请注册Stripe呢？大哥有这方面的经验不？

**Comment by jiangyangcreate** [comment]
周末去香港试一下

**Reply by Mistsink** [reply]
好滴！坐等大哥好消息呢！
